dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -39,13 +39,15 @@ encrypt.desc=Indicates whether the audit records in the audit file log will be e
 encryptAlias.name=Alias name of certificate for encrypting
 encryptAlias.desc=Alias name of the certificate used to encrypt the audit records.
 
-encryptKeyStoreRef.name=Audit encryption keystore reference 
+encryptKeyStoreRef.name=Audit Encryption Keystore Reference 
+encryptKeyStoreRef.name$Ref=Audit encryption keystore reference
```

Comment:
suggested edit:

Location of the keystore that contains the certificate that is used to encrypt the audit records.